File: php/samples/SimpleOrm.Sample/src/Migrations/V0006.php
<?php

declare(strict_types=1);

namespace SimpleOrm\Sample\Migrations;

use SimpleOrm\Migrations\MigrationVersion;
use SimpleOrm\Migrations\VersionBuilder;
use SimpleOrm\Sample\Migrations\View\UserTransactionTotal\V0006_AddLastTransactionAt;

final class V0006 extends MigrationVersion
{
    public function compose(VersionBuilder $version): void
    {
        $version
            ->apply(V0006_AddLastTransactionAt::class);
    }
}

File: go/orm/sample/migrations/V0006.go
package migrations

import (
	"github.com/kosmassp/SimpleOrm/go/orm"
	"github.com/kosmassp/SimpleOrm/go/orm/sample/migrations/View/UserTransactionTotal"
)

type V0006 struct{}

func (V0006) Compose(version *orm.VersionBuilder) {
	version.Apply(usertransactiontotal.V0006_AddLastTransactionAt{})
}
